### ISSUE TITLE
BUG: Errorhandling of communication between `consumedir` and `restserver`

### ISSUE THREAD

**Issue by totti4ever** [eikek/docspell#386]
I had an `$` in my `DOCSPELL_HEADER_VALUE` wich apparently breaks something (echo to console si not the problem, so it might be the configuration module?)

Nevertheless, what I think should definitively be fixed is that all consumed files failed the check:
```
- Result from checkfile: failed
- Checking file failed, skipping the file.
-> Some uploads failed.
```

While the curl call states `Forbidden` 

From the shown message I didn't really get that there was an authentication error, especially because the file was skipped, which would only make sense when there was an error with the file not with the setup

Long story short: I'd suggest to either add the curl response as information or make an if clause for the forbidden response and echo that apparently the setup (header value) didn't work out?!

**Comment by eikek** [comment]
Yes, definitely! If the error is known to be a "global" problem, then print the failure and quit immediately.
